controllers/configurationpolicy_controller.go

```diff
@@ -1880,6 +1880,35 @@ func (r *ConfigurationPolicyReconciler) determineDesiredObjects(
 			}
 		}
 
+		// If no objects were matched, return a compliant event since this is
+		// mustnothave and no objects were found.
+		if len(targetedObjects) == 0 {
+			var namespaces []string
+			for ns := range relevantNsNames {
+				namespaces = append(namespaces, ns)
+			}
```

Comment:
This is basically pulling all the namespaces from namespace/namespaceSelector. I realized I'm missing populating relatedObjects. I'll check to see what mustnothave with unnamed objects does and see whether it makes sense to do that here also.